src/TestFramework/TestFramework/Attributes/TestMethod/RetryBaseAttribute.cs

```diff
@@ -21,6 +21,7 @@ public abstract class RetryBaseAttribute : Attribute
     /// The other results are currently not used, but may be used in the future for tooling to show the
     /// state of the failed attempts.
     /// </returns>
+    [Experimental("MSTESTEXP", UrlFormat = "https://aka.ms/mstest/diagnostics#{0}")]
```

Comment:
This redirects to https://learn.microsoft.com/en-us/dotnet/core/testing/mstest-analyzers/overview, are we going to documentation MSTESTEXP there (even though it's not an analyzer)?

Reply:
I was wondering whether to have a separate page and decided it was probably "a kind of analyzer". I am doing the additional section as we speak.